test/forms/behaviours/StringFieldBehaviours.scala

```diff
@@ -70,6 +70,18 @@ trait StringFieldBehaviours extends FieldBehaviours {
       }
     }
 
+  def nameFieldWithInvalidCharacters(form: Form[_], fieldName: String, error: FormError): Unit =
+    "must not bind strings with invalid characters" in {
+
+      val generator: Gen[String] = RegexpGen.from(s"[!£^*(){}_+=:;|`~<>,±üçñèé]{$length}")
+
+      forAll(generator) {
+        invalidString =>
+          val result: Field = form.bind(Map(fieldName -> invalidString)).apply(fieldName)
+          result.errors must contain(error)
+      }
+    }
+
```

Comment:
This looks to be identical to the `fieldWithInvalidCharacters` method?

Reply:
Yes, because we do allow @ character in the input field, we needed a regex generator string without the @ character. There is probably a better way to do this, but that's what we came up with for now to avoid flaky tests.

Reply:
@justega247 the `traderDetails.holderOfTransit.name.error.invalid` error message has no mention of the @ character

Reply:
I would consider removing this method and just stripping out the @ sign from the other method, otherwise we could end up with loads of methods of various permutations of illegal characters! The ones that are there cover a good enough range

Reply:
Alright, will do that.